plugins/de.ovgu.featureide.core.featurehouse/src/de/ovgu/featureide/featurehouse/FeatureHouseComposer.java

```diff
@@ -1210,4 +1211,41 @@ public boolean needColor() {
 		return true;
 	}
 
+	@Override
+	public boolean supportsPartialFeatureProject() {
+		return true;
+	}
+
+	@Override
+	public void buildPartialFeatureProjectAssets(IFolder sourceFolder, ArrayList<String> removedFeatures, ArrayList<String> mandatoryFeatures)
+			throws IOException, CoreException {
+		deleteFeatureFolders(sourceFolder, removedFeatures);
+	}
+
+	private void deleteFeatureFolders(IFolder sourceFolder, ArrayList<String> removedFeatures) {
```

Comment:
Seems to complicated. Is it necessary to create the intermediate lists?

Reply:
You're right, it's unnecessary.